tests/test_result_to_dict.py

```diff
@@ -35,6 +35,13 @@ class Data(Base):
     data_enum = Column(Enum(MyEnum))
 
 
+def convert_str_to_dict(result):

```

Comment:
Lets make this call back do some nasty stuff 
for example. 
if this is our source object 
{
    object: '{"value": "JSON in Python", "number": 123}',
    nested_1: {
      nested_2: {
           nested_3: '{"KEY": "SOME VALUE", "object": '{"SOME_KEY": "VAL"}'}'
       }
    }
}

we can convert 
- `object`
- `nested_1.nexted_2.nexted_3`

TELL ME IF THIS WORKS:
 - `nested_1.nexted_2.nexted_3..object` (call back will convert `nested_3`, and then convert the content of `object_3`)

Reply:
also try to challenge the callback.